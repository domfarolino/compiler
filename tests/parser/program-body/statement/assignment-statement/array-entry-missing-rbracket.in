program dominic is

  integer index;
  integer dominic [10 : 20];

begin

  dominic [index := ;

end program.
