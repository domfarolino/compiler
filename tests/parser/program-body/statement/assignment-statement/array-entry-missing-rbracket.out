[1;31mLine 8: Expected ']' after expression in assignment destination, but got: ':='
Line 8: Error parsing program body
[0m